tests/test_conduit_cli.py

```diff
@@ -0,0 +1,197 @@
+import json
+from unittest.mock import patch
+
+import boto3
+from click.testing import CliRunner
+from freezegun import freeze_time
+from moto import mock_ec2
+from moto import mock_ecs
+from moto import mock_resourcegroupstaggingapi
+from moto import mock_secretsmanager
+from moto import mock_sts
+from moto.ec2 import utils as ec2_utils
+
+from commands.conduit_cli import create_task
+from commands.conduit_cli import exec_into_task
+from commands.conduit_cli import get_cluster_arn
+from commands.conduit_cli import get_postgres_secret_arn
+from commands.conduit_cli import is_task_running
+from commands.conduit_cli import tunnel
+
+
+@mock_ecs
+@mock_resourcegroupstaggingapi
+def test_get_cluster_arn(alias_session):
+    """Test that, given app and environment strings, get_cluster_arn returns the
+    arn of a cluster tagged with these strings."""
+
+    arn = boto3.client("ecs").create_cluster(
+        tags=[
+            {"key": "copilot-application", "value": "dbt-app"},
+            {"key": "copilot-environment", "value": "staging"},
+            {"key": "aws:cloudformation:logical-id", "value": "Cluster"},
+        ]
+    )["cluster"]["clusterArn"]
+
+    assert get_cluster_arn("dbt-app", "staging") == arn
```

Comment:
This would be easier to understand if it was broken into two stages...
```suggestion
    mocked_cluster = boto3.client("ecs").create_cluster(
        tags=[
            {"key": "copilot-application", "value": "dbt-app"},
            {"key": "copilot-environment", "value": "staging"},
            {"key": "aws:cloudformation:logical-id", "value": "Cluster"},
        ]
    )
    expected_arn = mocked_cluster["cluster"]["clusterArn"]

    assert get_cluster_arn("dbt-app", "staging") == expected_arn
```